packages/manager/apps/dedicated/client/app/ip/ip-range.service.js

```diff
@@ -35,11 +35,10 @@ angular.module('Module.ip.services').service('IpRange', function IpRange() {
   function getStrCopy(str, _copies) {
     const copies = _copies > 0 ? _copies : 1;
 
-    return times(copies + 1, str).join('');
+    return times(copies, () => str).join('');
```

Comment:
What was the issue there ? Lodash conversion ?

Reply:
Yes, related to Lodash conversion when importing the `ovh-manager-dedicated` repository into `manager` repository.